tools/liveimage-mount

```diff
@@ -700,19 +786,22 @@ def main():
         a = os.path.join(destmnt, 'run', 'initramfs', 'livedev')
         if os.path.lexists(a):
             os.unlink(a)
-        os.symlink(findmnt('-nro SOURCE -T ' + liveosmnt), a)
+        os.symlink(findmnt('-nro SOURCE -T', liveosmnt), a)
 
         mode = "'" + liveos + "' filesystems "
-        if tmpoverlayloop:
+        if tmpoverlayloop or tmpovl:
             mode += 'are only temporary.\n  Changes will NOT persist'
         else:
             mode += 'WILL persist'
         mode += ' after rebooting.'
 
+        a = X
+        if overlayfs:
+            a = Y
         environ = os.environ.copy()
-        if X:
+        if a:
             # This is overridden if PS1 is set in /root/.bashrc.
-            environ['PS1'] = X + '-[\u@\h \W]\$ '
+            environ['PS1'] = a.encode('utf_8') + b'-[\u@\h \W]\$ '
```

Comment:
The reason for that is that editors read `utf-8` but don't necessarily recognize `utf_8`.